test/server/proxy-test.js

```diff
@@ -806,6 +806,21 @@ describe('Proxy', () => {
                 });
             });
         });
+
+        it('Should generate an error if an error occurred when attempting to contact with destination server', done => {
```

Comment:
`Should respond with error if an error is occurred on attempting to connect with destination server`